test/Microsoft_Tests/src/SQLServer_Spec.enso

```diff
@@ -8,12 +8,15 @@ from Standard.Table.Errors import Invalid_Column_Names, Inexact_Type_Coercion, D
 
 import Standard.Database.DB_Column.DB_Column
 import Standard.Database.DB_Table.DB_Table
+import Standard.Database.Feature.Feature
 import Standard.Database.SQL_Type.SQL_Type
 import Standard.Database.Internal.Replace_Params.Replace_Params
 from Standard.Database import all
 from Standard.Database.Errors import all
 
 from Standard.Microsoft import all
+import Standard.Microsoft.Internal.SQLServer_Dialect
+from Standard.Microsoft.Internal.SQLServer_Dialect import all
```

Comment:
Can we be explicit about what we are importing?

Won't we clash on SQLServer_Dialect the module and the type?

Reply:
Namespaced import `SQLServer_Dialect.sqlserver`
Import everything un namespaced.

Typo on type name.

Reply:
`import Standard.Microsoft.Internal.SQLServer_Dialect as SQLServer_Dialect_Module`

Reply:
Fixed typo and removed immports